dask/blockwise.py

```diff
@@ -272,7 +271,7 @@ def __iter__(self):
         return iter(self._dict)
 
     def __len__(self):
-        return int(np.prod(list(self._out_numblocks().values())))
```

Comment:
Is this to drop NumPy as a dependency for this module, or for something else?

Reply:
Yes. It makes blockwise usable for dask.bag, which is something (given enough user interest) would be very beneficial e.g. to speed up ``db.Bag.map``.